content/en/security/application_security/getting_started/serverless.md

```diff
@@ -0,0 +1,147 @@
+---
+title: AWS Lambda Getting Started with ASM
+kind: documentation
+is_beta: true
+code_lang: serverless
+type: multi-code-lang
+code_lang_weight: 90
+further_reading:
+    - link: "/security/application_security/how-appsec-works/"
+      tag: "Documentation"
+      text: "How Application Security Works"
+    - link: "/security/default_rules/#cat-application-security"
+      tag: "Documentation"
+      text: "OOTB Application Security Management Rules"
+    - link: "/security/application_security/troubleshooting"
+      tag: "Documentation"
+      text: "Troubleshooting Application Security Management"
+    - link: "/security/application_security/threats/"
+      tag: "Documentation"
+      text: "Application Threat Monitoring and Protection"
+    - link: "/security/application_security/risk_management/"
+      tag: "Documentation"
+      text: "Application Risk Management"
+---
+
+{{< callout url="#" header="ASM support for AWS Lambda is in beta" btn_hidden="true" >}}
```

Comment:
This note formatting is different than the one for Risk Management. Should we re-use the same format so it's standard everywhere?

Reply:
https://docs.datadoghq.com/security/application_security/risk_management/